port/aggregation-properties/resource_test.go

```diff
@@ -0,0 +1,787 @@
+package aggregation_properties_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/port-labs/terraform-provider-port-labs/internal/acctest"
+	"github.com/port-labs/terraform-provider-port-labs/internal/utils"
+)
+
+func TestAccPortAggregationPropertyWithCycleRelation(t *testing.T) {
+	// Test checks that a cycle aggregation property works.
+	// The cycle is created by creating a parent blueprint and a child blueprint.
+	// The child blueprint has a relation to the parent blueprint.
+	// The parent blueprint has an aggregation property that counts the children of the parent.
+	// The aggregation property is created with a cycle relation to the child blueprint, which is allowed.
+	parentBlueprintIdentifier := utils.GenID()
+	childBlueprintIdentifier := utils.GenID()
+	var testAccActionConfigCreate = fmt.Sprintf(`
+	resource "port_blueprint" "parent_blueprint" {
+		title = "Parent Blueprint"
+		icon = "Terraform"
+		identifier = "%s"
+		description = ""
+		properties = {
+			number_props = {
+				"age" = {
+					title = "Age"
+				}
+			}
+		}
+	}
+
+	resource "port_blueprint" "child_blueprint" {
+		title = "Child Blueprint"
+		icon = "Terraform"
+		identifier = "%s"
+		description = ""
+       	relations = {
```

Comment:
Something seems not right with the identation of relations throughout the file, it seems like because here the indentation is with spaces, and other places with tabs